sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -164,8 +164,7 @@ object Klaviyo {
     }
 
     /**
-     * Updates the currently tracked profile from a map-like [Profile] object
-     * Saves identifying attributes (externalId, email, phone) to the currently tracked profile.
+     * Replaces the currently tracked profile with a new [Profile] object
```

Comment:
Maybe re-iterating that this will reset the existing profile.

Reply:
good thinking, added an @see for reset profile